cdc/owner.go

```diff
@@ -750,6 +754,9 @@ func (o *Owner) flushChangeFeedInfos(ctx context.Context) error {
 		}
 	}
 	if time.Since(o.gcSafepointLastUpdate) > GCSafepointUpdateInterval {
+		if (time.Now().Unix() - int64(minCheckpointTs)) > o.gcBLT {
+			minCheckpointTs = uint64(time.Now().Unix() - o.gcBLT)
```

Comment:
the name of `minCheckpointTs` is not accurate any more, may be we can call it `gcSafePoint`

Reply:
ok, I will change it.

Reply:
`gcSafePoint` or `gcSafepoint`, keep all the same...